Dockerfile.ubi

```diff
@@ -73,6 +73,9 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 ## Development #################################################################
 FROM python-cuda-base AS dev
 
+RUN --mount=type=cache,target=/root/.cache/uv \
+    uv pip install --no-build-isolation "git+https://github.com/state-spaces/mamba@v2.2.4"
```

Comment:
Moving this step to a separate layer, taking care of pre-installing build dependencies and setting the correct build env vars might work without having to change subsequent steps since `uv` will cache the built wheel and won't try to download it.

For reference on how to build mamba-ssm: https://github.com/neuralmagic/nm-cicd/pull/122/files

Reply:
done